src/utils/fastapi.py

```
@@ -10,28 +10,32 @@
 
 LOGIN_KEY = "bot:jwt"
 
-@app.get("/status")
-async def status():
+
+async def get_logged_in() -> bool:
     async with httpx.AsyncClient() as client:
         jwt = r.get(LOGIN_KEY)
-        if not jwt: 
-            return {"logged_in": False}
+        if not jwt: return False
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)